share/picongpu/tests/CollisionsBeamRelaxation/include/picongpu/param/memory.param

```diff
@@ -102,7 +102,7 @@ namespace picongpu
     };
 
     /** number of scalar fields that are reserved as temporary fields */
-    constexpr uint32_t fieldTmpNumSlots = 1;
+    constexpr uint32_t fieldTmpNumSlots = 3;
```

Comment:
Why do we need 3 slots?

If one of the collision parts requires 3 solts please do not forget to add a static assert to the code snipped to guard against user configuration mistakes.

```
PMACC_CASSERT_MSG(_please_allocate_at_least_three_FieldTmp_in_memory_param, fieldTmpNumSlots >= 3);
```